examples/combined-hbs-jsx/views/Home.jsx

```diff
@@ -0,0 +1,34 @@
+import fs from 'fs'
+import Handlebars from 'handlebars'
+import path from 'path'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+import Component from './components/Component'
+
+export default function Home({ pageTitle }) {
+  function createMarkup(markup) {
+    return { __html: markup }
+  }
+
+  const fileContent = String(
+    fs.readFileSync(path.join(__dirname, './partials/partial-with-jsx-component.hbs'))
```

Comment:
two comments about this:

- instead of coercing it to `String`, you could use the second argument of `readFileSync` to set the encoding to `utf8` and return it as a string not a buffer
- this will block the event loop. fine for a one-off example, but people will probably copy/paste it, so best to avoid something that would cause performance problems in a production app. it could be pulled out of the component to the top of the file to avoid this

Reply:
Fixed in this [commit](https://github.com/Financial-Times/dotcom-page-kit/pull/714/commits/19bbc092bf60d4762205b8352068df0609442d98).